server/repository/src/db_diesel/clinician_link_row.rs

```diff
@@ -106,8 +106,9 @@ impl<'a> ClinicianLinkRowRepository<'a> {
 }
 
 impl Upsert for ClinicianLinkRow {
-    fn upsert_sync(&self, con: &StorageConnection) -> Result<(), RepositoryError> {
-        ClinicianLinkRowRepository::new(con).upsert_one(self)
+    fn upsert(&self, con: &StorageConnection) -> Result<Option<i64>, RepositoryError> {
+        ClinicianLinkRowRepository::new(con).upsert_one(self)?;
+        Ok(None) // Table not in Changelog
```

Comment:
Added this everywhere I found a `Upsert` without a changelog.
Hopefully easier to find that way? With OMS Central we'll want changelog for all these tables I think.

Reply:
Thanks !